.github/workflows/label-linter.yml

```diff
@@ -0,0 +1,40 @@
+name: "Lint Labels"
+
+on: [pull_request]
+
+jobs:
+  label-linter:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+
+      # The Add Labels workflow will apply a label based on the
+      # paths that are modified in the pull request.
+      #
+      # To use this workflow, you will need to set up a .github/label-ruleset.yml
+      # file with configuration.  For more information, see:
```

Comment:
Update first sentence: 
To use this workflow, you must set up and configure a .github/label-ruleset.yml file.

Reply:
Done.